graftm/bootstrapper.py

```diff
@@ -26,34 +28,48 @@ def __init__(self, **kwargs):
         self.evalue = kwargs.pop('evalue',None)
         self.min_orf_length = kwargs.pop('min_orf_length',None)
         graftm_package = kwargs.pop('graftm_package',None)
```

Comment:
this is my code, but we should check kwargs to make sure there is nothing extra being ignored, like in `Create`